core/src/main/java/org/testcontainers/containers/Network.java

```diff
@@ -1,21 +1,27 @@
 package org.testcontainers.containers;
 
 import com.github.dockerjava.api.command.CreateNetworkCmd;
-import lombok.Builder;
-import lombok.Getter;
-import lombok.Singular;
+import lombok.*;
 import org.junit.rules.ExternalResource;
 import org.junit.rules.TestRule;
 import org.testcontainers.DockerClientFactory;
 import org.testcontainers.utility.ResourceReaper;
 
+import java.util.Collections;
 import java.util.Set;
 import java.util.UUID;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.function.Consumer;
 
 public interface Network extends AutoCloseable, TestRule {
 
+    Network SHARED = new NetworkImpl(false, null, Collections.emptySet(), null) {
+        @Override
+        public void close() {
+            // Do not avoid users to close SHARED network, only ResourceReaper is allowed to close (destroy) it
```

Comment:
I think you meant _allow_ instead of _avoid_.

Reply:
snap! :D